controllers/v1/admin.js

```diff
@@ -123,4 +123,62 @@ module.exports = class Admin {
 			}
 		})
 	}
+
+	/**
+   * @api {post} /project/v1/admin/deleteResource/:resourceId?type=solution
+   * Deletes a resource (program/solution) after validating admin access.
+   * @apiVersion 1.0.0
+   * @apiGroup Admin
+   * @apiSampleRequest /project/v1/admin/deleteResource/683867e60f8595db9c1b6c26?type=solution
+    * @apiParamExample {json} Response:
+    {
+    "message": "Solution and associated resources deleted successfully",
+    "status": 200,
+    "result": {
+        "solutionDeletedCount": 1,
```

Comment:
@MallanagoudaB can we  change this variable name to deletedSolutionsCount, deletedProjectTemplatesCount etc. existing set not sound grammatically correct